NBitcoin/Network.cs

```diff
@@ -1459,13 +1459,41 @@ public uint Magic
 			}
 		}
 
+		private bool isBTCPForkEnabled(int nHeight)
+		{
+			return nHeight > 272991; // nForkStartHeight
+		}
+
+		// TODO - use
+		public Money GetRewardBTCP(int nHeight)

```

Comment:
Remove this, no altcoin specific logic in `Network.cs`.

Reply:
Done